Makefile

```diff
@@ -84,7 +87,7 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 
 .PHONY: test
 test: manifests kustomize generate fmt vet envtest ## Run tests.
-	. ./testing/set-env-vars.sh; go test ./... -timeout 30s -coverprofile cover.out -v
+	. ./testing/set-env-vars.sh; go test ./... -timeout $${SUITE_TIMEOUT} -coverprofile cover.out -v
```

Comment:
Add a default here, and think about removing that Env variable from the top of make file.